src/Commands/Modules/Utility/MathCommand.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace BrackeysBot.Commands
+{
+    using System.Drawing;
+    using System.Drawing.Drawing2D;
+    using System.Drawing.Imaging;
+    using System.IO;
+    using System.Threading.Tasks;
+    using CSharpMath.SkiaSharp;
+    using Discord.Commands;
+    using SkiaSharp;
+
+    public partial class UtilityModule : BrackeysBotModule
+    {
+        [Command("math"), Alias("maths", "eq", "equation", "tex", "latex")]
+        [Summary("Renders LaTeX input to an image, and attaches the result.")]
+        public async Task MathCommandAsync([Summary("The LaTeX input."), Remainder]
+                                           string input)
+        {
+            using var originalImage = Render(input);
+            using var image = AddPadding(originalImage, 20);
+            using var resized = Resize(image, image.Size / 2);
```

Comment:
Is there a reason to resize the image here? Is it possible to already render the equation at a lower resolution to save processing power?

Reply:
I will have a look through the documentation of CSharpMath and see what I can do. I admit it was a rather hacky solution to resize this way.

Reply:
Alright, resolved with 8f75bb4bc05cd6f84cb0fc80e26548b343e59852

The MathPainter object has a FontSize property, which defaults to 50. Now explicitly set to 25. Come to think of it, it would probably be better served through a config value...

Reply:
Nice and elegant fix! I don't think a config value is necessary, since there's no reason for us to pretty much _ever_ change the size.